### ISSUE TITLE
"PyCUDA was compiled without GL extension support"

### ISSUE THREAD

**Issue by booker-max** [JOP-Lee/READ#48]
when I run the "python viewer.py --config downloads/kitti6.yaml",
I meet the problem, "pycuda import error: PyCUDA was compiled without GL extension support", I don't know the reason, 
Can you help me, I'm sure that I follow the requirements.sh to install the environment

**Comment by yixionghuo** [comment]
Hi, I had the same problem as you, did you solve it?

**Comment by booker-max** [comment]
yes, i solved it, you only need to run the code in /src/scripts/train_data.sh.
